skins/cat17/templates/Donation_Form.html.twig

```diff
@@ -53,6 +53,8 @@
 								<div class="wrap-amount-typed field-grp" data-currency="€">
 									<input type="text" id="amount-typed">
 								</div>
+								<span class="error-text">{$ 'form_amount_error' | trans $}</span>
```

Comment:
Key still seems to be missing from the content repo.

Reply:
Added in https://github.com/wmde/fundraising-frontend-content/pull/84